src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -0,0 +1,190 @@
+import { guid } from "../../utils/guid";
+import { chevronLeft16 } from "@esri/calcite-ui-icons";
+import CalciteIcon from "../../utils/CalciteIcon";
+
+@Component({
+  tag: "calcite-accordion-item",
+  styleUrl: "calcite-accordion-item.scss",
+  shadow: true
+})
+export class CalciteAccordionItem {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  @Prop({ reflect: true, mutable: true }) active: boolean = false;
+
+  /** pass a title for the accordion item */
+  @Prop() itemTitle: string;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  //--------------------------------------------------------------------------
+
+  @Event() calciteAccordionItemKeyEvent: EventEmitter;
+  @Event() calciteAccordionItemSelected: EventEmitter;
+  @Event() closeCalciteAccordionItem: EventEmitter;
+  @Event() registerCalciteAccordionItem: EventEmitter;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  //--------------------------------------------------------------------------
+
+  componentDidLoad() {
+    this.itemPosition = this.getItemPosition();
+    this.registerCalciteAccordionItem.emit({
+      item: this.el,
+      position: this.itemPosition
+    });
+  }
+
+  render() {
+    const dir = getElementDir(this.el);
+    const theme = getElementTheme(this.el);
+    const scale = getElementProp(this.el, "scale", "m");
+    const appearance = getElementProp(this.el, "appearance", "default");
+    const iconPosition = getElementProp(this.el, "icon-position", "end");
+    return (
+      <Host
+        theme={theme}
+        dir={dir}
+        scale={scale}
+        appearance={appearance}
+        icon-position={iconPosition}
+        id={this.accordionItemId}
+        tabindex="0"
+        role="menuitem"
+        aria-expanded={this.active ? "true" : "false"}
+      >
+        <div
+          class="accordion-item-header"
+          onClick={() => this.emitRequestedItem()}
+        >
+          <span class="accordion-item-title">{this.itemTitle}</span>
+          <div class="accordion-item-icon">
+            <CalciteIcon size="16" path={chevronLeft16} />
+          </div>
+        </div>
+        <div class="accordion-item-content">
+          <slot />
+        </div>
+      </Host>
+    );
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Event Listeners
+  //
+  //--------------------------------------------------------------------------
+
+  @Listen("keydown") keyDownHandler(e) {
+    switch (e.keyCode) {
```

Comment:
event.keyCode is deprecated. Use event.key instead. It's also easier to code because it prints key names, like "Enter" instead of a number.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
One weird gotcha is instead of "Space" it's a literal space " ";